driver/asana_driver.py

```diff
@@ -124,28 +138,34 @@ def retrieve_metadata(self, milestone):
         :returns `RetrieveMetadataResult` with the appropriate values populated.
         """
         
-        if milestone is None: milestone = { 'step':'stage1' } # TODO: verify use of milestone
+        if milestone is None: milestone = {  }
```

Comment:
No need to worry about milestone being `None`, should that be the case within your findings here let me know so I can default it.